<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicWebEntainRadioWidget</class>
 <widget class="QWidget" name="MusicWebEntainRadioWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络电台播放窗口</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>475</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>网络电台播放窗口</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>473</x>
     <y>22</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>161</y>
     <width>455</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="movieLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>55</y>
     <width>455</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>122</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MusicClickedSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>122</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="colletButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>122</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="discolletButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>122</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>130</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>音量</string>
   </property>
  </widget>
  <widget class="QLabel" name="stateLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>60</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="backgroundMask" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>475</width>
     <height>295</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>backgroundMask</zorder>
  <zorder>topTitleName</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>listWidget</zorder>
  <zorder>movieLabel</zorder>
  <zorder>playButton</zorder>
  <zorder>volumeSlider</zorder>
  <zorder>stopButton</zorder>
  <zorder>colletButton</zorder>
  <zorder>discolletButton</zorder>
  <zorder>label_2</zorder>
  <zorder>stateLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicClickedSlider</class>
   <extends>QSlider</extends>
   <header>musicclickedslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
